less/components/best-offers/best-offers.less

```diff
@@ -1,11 +1,6 @@
 #best-offers{
-  .nav-search{
+  .serach-vehicles{
```

Comment:
Change the class name to ".search-vehicles" like <div class="search-vehicles">, so this way this can work correctly.